Update onedrive-lti.md

msbemba opened the pull request:
Added another step after 9 stating that the remaining permission are needed. so another admin consent is required.

msbemba committed:
Update onedrive-lti.md

Added another step after 9 stating that the remaining permission are needed. so another admin consent is required.

msbemba commented:
@JohanFreelancer9  Please copy review

msbemba commented:
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7686

JohanFreelancer9 commented on microsoft-365/lti/onedrive-lti.md:
```diff
@@ -70,7 +70,8 @@ Integrating Microsoft OneDrive LTI with Canvas is a two step process. The first
 
 9. Review and save your changes. A message will be displayed on successful registration.
-10. Your registration details can also be reviewed by selecting the **View LTI Tenants** button on the home page.
+10. Sign into the <a href="https://onedrivelti.microsoft.com/admin" target="_blank">Microsoft OneDrive LTI Registration Portal</a> and  Select the **Admin Consent** button and accept the remaining permissions.
```
```suggestion
10. Sign into the <a href="https://onedrivelti.microsoft.com/admin" target="_blank">Microsoft OneDrive LTI Registration Portal</a>, select the **Admin Consent** button, and accept the remaining permissions.
```

JohanFreelancer9 reviewed (changes requested):
Copy review.

msbemba committed:
Update microsoft-365/lti/onedrive-lti.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

yogkumgit commented:
@MicrosoftHeidi Please merge this PR. Thanks

yogkumgit commented:
@MicrosoftHeidi Gentle reminder.

yogkumgit commented:
@MicrosoftHeidi Kindly merge this PR. Thanks

DaniEASmith committed:
Update onedrive-lti.md

Adding end-user article link and notice that steps 1 & 2 for admin consent may need to be repeated with future releases.

DaniEASmith committed:
Merge branch 'public' into patch-944
